show/main.py

```diff
@@ -187,6 +186,38 @@ def summary(interfacename):
         command = cmd_ifconfig
         run_command(command)
 
+
+@interfaces.group(cls=AliasedGroup, default_if_no_args=True)
+def transceiver():
```

Comment:
`transciever` subcommand is redundant with `sfp` command on line 250. Please remove `sfp` subcommand.

Reply:
done -- but please look closely to my next patch in this section as i haven't been able to test this command due to my dut platform not being fully (sfputil) complaint yet.

Reply:
Looks good to me.